tools/PowerShell/Microsoft.WinGet.Client/src/Microsoft.WinGet.Client.psm1

```diff
@@ -0,0 +1,16 @@
+. $PSScriptRoot\Library\Get-WinGetVersion.ps1
+
+$RequiredVersion = "1.1.12653"
+
+$version = Get-WinGetVersion
+if ($version -lt $RequiredVersion) {
+	# Need to do localization
```

Comment:
>	 [](http://example.com/codeflow?start=0&length=1)

nit: all these files are using spaces but line 7,9 and 15 use tabs

Reply:
switched